### ISSUE TITLE
Support RAML v1.0

### ISSUE THREAD

**Issue by econchick** [spotify/ramlfications#54]
Since RAML v1.0 [is coming](https://github.com/raml-org/raml-spec/milestones/v1.0) with the goal of at the [end of October](https://github.com/raml-org/raml-spec/milestones), this issue will be a collection of all changes (that may or may not need to be addressed in `ramlfications`) - to be updated in the comments.  

No guarantee that `ramlfications` will be updated by when the new spec comes out, just FYI :) This is more for folks coming here wondering if this is being worked on or not.

**Comment by pmackay** [comment]
There are 7 changes listed [here](http://docs.raml.org/specs/1.0/#raml-10-spec-what-39-s-new-and-different-in-raml-1-0) - would it make sense to create new issues for each one?

**Comment by econchick** [comment]
@pmackay yes, but I'm actually in the middle of making a roadmap to put on the (currently empty) wiki, of which more detailed issues will come from, if that makes sense.

**Comment by econchick** [comment]
Currently scoping this out in [the wiki](https://github.com/spotify/ramlfications/wiki/RAML-v1.0-Roadmap)!

**Comment by j-funk** [comment]
@econchick ooc is this still a priority?  I saw the wiki link above was empty.

**Comment by econchick** [comment]
hey @j-funk ! It's still a priority, but development has stalled because I moved to NYC a couple of weeks ago for a new job (same company). So I've been distracted with getting settled in. I guess this is part of the problem, when there's only one main developer :-! But I'm spending some time this weekend to get back up to speed.

**Comment by mcm** [comment]
@econchick Has any work started on RAML 1.0? I'm working with another developer on laying the foundations for all of our internal API's going forward, and I think it would make more sense to add RAML 1.0 support to ramlfications than to write our own separate library. Just want to make sure I start in the right place if any work has begun.

**Comment by jamespharaoh** [comment]
i also need this and will be looking to implement it myself, if necessary, so please anyone share what you have. initially i need to be able to load raml and create documentation from it.

**Comment by econchick** [comment]
@jamespharaoh take a look at the other library I've released, [griffin](https://github.com/spotify/griffin). It creates HTML/static docs from RAML files.

**Comment by econchick** [comment]
@mcm the v0.2.0-dev branch is the current progress of RAML1.0 support. It's started, but it's slow. I'd hate for you to write something separately. I'd love for any help, which is outlined on [the wiki](https://github.com/spotify/ramlfications/wiki/RAMLfications-v0.2.0-Release)

Feel free to hop into the #ramlfications slack channel at slackin.spotify.com, or on IRC, if you want to chat.

**Comment by j-funk** [comment]
@econchick you might want to update the earlier link to the empty wiki at the top of this thread as well :)

**Comment by econchick** [comment]
@j-funk thanks! I just did

**Comment by sichvoge** [comment]
@econchick is there any specific timeline you can share for the RAML 1.0 support?

**Comment by econchick** [comment]
Hey @sichvoge - I wish I could; it's pretty much down to just [unpaid] work on the weekends for me so it's pretty slow, and there have not been any recent contributions from others to help. I understand there's a lot of interest to get support for 1.0.  Perhaps I should run a kickstarter or something to help give motivation and negate the issue of work going unpaid (sorta kidding 😉 ).

**Comment by sichvoge** [comment]
HAHA :D Let me know if you need more advertisement. I am planning to add an article to the "Sneak Peek" section in the RAML forums and consolidate infos about the current tools that are in dev. Maybe there is a good chance to ask for help ;)

**Comment by sichvoge** [comment]
Not sure if you are aware but you can try out the following [World Music Example](https://github.com/raml-org/raml-examples/blob/master/others/world-music-api/api.raml) to test what works and what isn't working yet. 

The example contains 90% of all features excluding overlays and extensions.

**Comment by econchick** [comment]
@sichvoge this is amazing, thank you! I'd love any other full examples like this, if you come across them/publish.

**Comment by sichvoge** [comment]
you see all i have in the same repo ;)

**Comment by ptxmac** [comment]
Any update on this?

**Comment by EpocDotFr** [comment]
Looks like this repo is being abandonned.

**Comment by econchick** [comment]
@EpocDotFr it has not - I've been slowly working on 1.0 support in the dev branch, with a sprint held at PyCon this year. There's been a lot of conference travel, but late July & August, development should pick back up.

**Comment by EpocDotFr** [comment]
@econchick My bad. I didn't check the dev branch.

**Comment by acs** [comment]
Th development seems to be stalled for 2 years now. @econchick do you know of any alternative to work with RAML 1.0 in Python? 

Cheers!

**Comment by eduardo-rubio** [comment]
Hi @econchick Could you please give us an update about the RAML 1.0 support? BTW, have you seen this repo https://github.com/raml-org/webapi-parser? Maybe it could help you to speed up your development. Regards!

**Comment by econchick** [comment]
hey @eduardo-rubio et al - I am not able to give this project the attention it deserves. I've opened up a new issue soliciting for maintainers: #162
